index.src.html

```diff
@@ -54,6 +56,15 @@ <h1>Clear Site Data</h1>
 spec: PSL; urlPrefix: https://publicsuffix.org/list/
   type: dfn
     text: registered domain; url: #
+spec: html; urlPrefix: https://html.spec.whatwg.org/
+  type: dfn
+    text: document state; url: #document-state-2
+spec: html; urlPrefix: https://html.spec.whatwg.org/
+  type: dfn
+    text: fully active; url: #fully-active
```

Comment:
This is not necessary. Just reference the term properly, using `[=Document/fully active=]` instead of `<a>fully active</a>`.